packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -315,6 +314,27 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
          * Show Opened File in New Window	workbench.action.files.showOpenedFileInNewWindow
          * Compare Opened File With	workbench.files.action.compareFileWith
          */
+
+        // Register built-in language service commands
+        // see https://code.visualstudio.com/api/references/commands
+        // tslint:disable: no-any
+        commands.registerCommand(
+            {
+                id: 'vscode.executeDocumentSymbolProvider'
+            },
+            {
+                execute: (resource: URI) => commands.executeCommand('monaco._executeDocumentSymbolProvider',
```

Comment:
just a question: why not use async/await instead of the callbacks ?

Reply:
sorry, I don't understand what you mean. Could you clarify?

Reply:
callback :
```
 commands.executeCommand(...).then((foo) => {return foo.something() })
```
vs

async/await:
```
 const foo = await commands.executeCommand(...)
return foo.something()
```

Reply:
Ah, I see. No specific reason but as this is a single expression I find it more readable and it has less code this way.